apps/a11y-tests/src/axe-sarif-converter/axe-utils.ts

```diff
@@ -0,0 +1,108 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
Don't think you need this comment at the beginning of each file, since we have a repo-wide license file.